main/res/layout/logcache_trackable_item.xml

```diff
@@ -73,4 +75,4 @@
         tools:text="action"
         android:maxLines="1" />
 
-</LinearLayout>
```

Comment:
These whitespace fixes (3 occurrences) are unrelated and should be removed from the commits (you can open a separate PR to tackle them).